src/pages/Dashboard/MintRepayVai/MintVai/index.spec.tsx

```diff
@@ -108,5 +118,32 @@ describe('pages/Dashboard/MintRepayVai/MintVai', () => {
     });
   });
 
+  it('asks the user to enable token if not enabled', async () => {
```

Comment:
I saw you added this test for the supply/withdraw modal too. I think we should add tests to the `EnableToken` component instead, so that we don't have to duplicate them on every component that uses it.